build.gradle

```diff
@@ -53,6 +53,10 @@ dependencies {
     implementation files('libs/industrialcraft-2-2.8.170-ex112-api.jar')
     implementation files('libs/NBT-Exporter-1.0-forgelib.jar')
 
+    // Optional mods
+    implementation "mezz.jei:jei_1.12.2:4.16.1.302"
+    implementation "mcjty.theoneprobe:TheOneProbe-1.12:1.12-1.4.28-17"
```

Comment:
No reason to add these directly unless we have code calling them. Not difficult to install them manually to the mods folder.

Reply:
will be removing those